src/components/Form/FormWrapper.tsx

```diff
@@ -63,7 +63,7 @@ function FormWrapper({
 }: FormWrapperProps) {
     const styles = useThemeStyles();
     const formRef = useRef<RNScrollView>(null);
-    const formContentRef = useRef<View>(null);
+    const formContentRef = useRef<RNView>(null);
```

Comment:
I think I'm missing something, what is the purpose of this change? 
Aren't `RNView` and `View` the same?

Reply:
Yes they're the same, I'm going to revert this!